src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -24,12 +27,33 @@ public ArticleController(ArticleRepository articleRepository) {
     }
 
     @PostMapping("/qna")
-    public String addUser(@ModelAttribute Article article) {
-        articleRepository.save(article);
-        log.debug("debug log={}", article.getContents());
+    public String addArticle(@RequestParam String title,
+                             @RequestParam String contents,
+                             Model model, HttpSession httpSession) {
+
+        if (httpSession.getAttribute(SessionConst.LOGIN_USERID) == null) {
```

Comment:
```suggestion
        if (httpSession.getAttribute(SessionConstant.LOGIN_USERID) == null) {
```
1) 클래스 이름은 줄이지 않는게 좋습니다.
2 필요하다면 static import를 활용하면 코드가 깔끔해 질 수 있습니다.